plugins/cost-insights/src/components/CostOverviewCard/CostOverviewBreakdownChart.tsx

```diff
@@ -186,10 +186,10 @@ export const CostOverviewBreakdownChart = ({
         ? DateTime.fromMillis(label)
         : DateTime.fromISO(label!);
     const dateTitle = date.toUTC().toFormat(DEFAULT_DATE_FORMAT);
-    const items = payload.map(p => ({
+    const items = payload.map((p, i) => ({
       label: p.dataKey as string,
-      value: formatGraphValue(p.value as number),
-      fill: p.fill!,
+      value: formatGraphValue(p.value as number, i),
+      fill: p.color!,
```

Comment:
I'm double-checking if `p.color` is the right attribute here since `fill` was removed from the `Payload` type.